apps/vth-frontend/src/components/Card/index.tsx

```diff
@@ -0,0 +1,28 @@
+import { Heading2, Paragraph } from '@utrecht/component-library-react';
+import clsx from 'clsx';
+import Image from 'next/image';
+import Link from 'next/link';
+import React, { FC, HTMLAttributes } from 'react';
+import './index.style.css';
+
+interface CardProps extends Omit<HTMLAttributes<HTMLDivElement>, 'children'> {
+  alt: string;
+  body: string;
+  href: string;
+  imgSrc: string;
```

Comment:
What do you think if we split the image and link properties in the following way?

```tsx

image: {
url: string
alt: string
}
link: {
href: string;
text:  string;
 }

```